### PR TITLE
Integrate LangChain into the project

### PR DESCRIPTION
**Pull request by kamka427**
### What is the purpose of this PR?

- Integrates LangChain in to the projekt to get AI generated answers.

### How did you verify your code is stable and works correctly?

- Call the api in Postman.

### CODE REVIEW

File: app/api/chat/route.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+If you can't answer a question, you should ask them to ask a real professional.
+Question: {question}}`;
+
+export async function POST(req: NextRequest) {
+  try {
+    const body = await req.json();
+    const chat = new ChatOpenAI({});
+    const prompt = PromptTemplate.fromTemplate(PROMPT_TEMPLATE);
+    const runnable = prompt.pipe(chat);
+    console.log(body.question);
```
Comment (MilosAlex) [inline app/api/chat/route.ts:19]: Is this log left here on purpose?

**Review by MilosAlex — DISMISSED** [review]

File: app/api/chat/route.ts (hunk 1 above)
Comment (kamka427) [reply app/api/chat/route.ts:19]: Thank you! Nice catch.

**Review by ssbenedek — APPROVED** [review]
Keep it up, not so much left

**Review by MilosAlex — APPROVED** [review]
Great job! Looking forward to see this in action!

**Review by ssbenedek — APPROVED** [review]

**Pull request merged**